data/successors/successor news.txt

```diff
@@ -0,0 +1,277 @@
+			`A proud-looking Successor naval officer stands before an imposing-looking warship: "Enlist!" the caption reads. "See the stars today!"`
+			`Several short vignettes play: a group of Successors unearthing ruins, the inside of a laboratory, a calm pastoral scene. A voiceover speaks: "Uncovering our heritage; bettering our lives." Meticulously-dressed officials of the High Houses are present in each situation, presented as supportive overseers.`
+			"A advertisement plays showing the sleek lines of a Successor warship, its polished armor gleaming in the sunlight. It goes on for several minutes, showing the vessel from numerous angles while a sweeping soundtrack plays."
+
+news "successor advertisements"
+	to show
+		has "language: Successor"
+	location
+		attributes successor
+		not attributes quiet uninhabited
+	name
+		word
+			"Advertisement"
+	message
+		word
+			"A bowl of a stringy kelp-like food dances around a screen while various seasonings are applied to it. After about thirty seconds, a Successor is shown slurping some of the strands, and their skin turns a happy pink."
+			"A cluster of small recreational sailing ships are shown on a pristine lake at sunset, their Successor crew either manning them or splashing happily in the water. A caption at the bottom encourages the viewer to purchase their own."
+			"An advertisement plays for the Successor Pulse Laser: it is shown being constructed in a clean, bright factory, installed on a ship, and extracting minerals from asteroids. It is never shown in a combat situation."
+			"An advertisement plays showing a variety of attractive and expensive-looking jewellery, made of gold, silver, and other stranger metals which you cannot recognize. A variety of models display it wrapped around their arms or other parts of their body, gently undulating as the camera moves."
+			"A display shows an advertisement for one of the Successor universities, showing elegant, sculptural buildings emerging from the ground under a rain-soaked sky."
+			"An field of large, floral plants are shown growing long strings of gleaming metal which are quickly harvested by agricultural robots."
+			"A series of images displaying several models flash one after another so quickly that it is hard to tell what is actually being advertised."
+			"A short video clip is shown advertising a performance in a local concert hall. The stage is divided into two sections: one is underwater and the other is dry."
+			"Some sort of interactive game that you can't make sense of captivates the attention of a group of young Successors."
+			"An eye covering vaguely resembling a contact lens is shown being applied to the eyes of several models, changing their color and shape when applied."
+			"A shipbuilding organization advertises a members' discount on Kvelqde shuttles. They make sure to note that their operation is supported by one of the High Houses."
```

Comment:
You probably don't _need_ to add this, but which High House is supporting their operation?

Reply:
The trouble is that specifying it narratively restricts where it could be offered, since it isn't likely that e.g. one of the High Houses is going to be advertising on the home planet of another.

Reply:
That's a good point. Maybe leave it as is, but possibly add a couple briefer news items linked to the most prominent High Houses that only offer on their home planets?